### DISCUSSION TITLE
nothing is happening

### DISCUSSION THREAD

**Discussion by blackulaphoto** [AntlerVC/firetable#1387] [Support / Q&A]
Hi so i watched a tutorial on how to use your storybook template and from what i could tell you just fill in the initial prompt and hit enter and everything gets created , well ive configured everything to the letter and filled in prompts but nothing happens , i dont know what im doin wrong can i get some help with this please

**Comment by harinij** [comment]
Could you please share some screenshot/recording with explanation of where you see the issue. It is unclear for us without reproduction steps how to further proceed here. Closing this for now, in case you still facing this issue, reopen by commenting with further detailed explanation including reproduction steps of the issue.
